packages/manager/modules/web-paas/src/components/additional-option/controller.js

```diff
@@ -0,0 +1,26 @@
+import get from 'lodash/get';
+
+export default class WebPaasProjectAdditionalOptionCtrl {
+  $onInit() {
+    this.disabled = false;
+  }
+
+  getPrice() {
+    const { price } = get(this.option, 'prices').find(({ capacities }) =>
```

Comment:
```suggestion
    const { price } = this.option?.prices.find(({ capacities }) =>
```
or 
```suggestion
    const { price } = get(this.option, 'prices', []).find(({ capacities }) =>
```
Here if prices is undefined, it will still crash on find